woocommerce/woocommerce-gateway-stripe
# feat: add payments settings section

# Changes proposed in this Pull Request:

Fixes https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1723

Adds the "Payments & Transactions" section, per newer designs.
This does not include the "preview" section for the statement descriptor.

![2021-09-06 17 07 50](https://user-images.githubusercontent.com/273592/132262943-f7a9b8ab-21e7-4045-aeb7-f1979e5f1378.gif)

# Testing instructions
- Ensure you have the `_wcstripe_feature_upe_settings` flag enabled
- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=stripe&panel=settings
- The new section appears
